students/phrost/Lesson04/Mailroom_Part_2.py

```diff
@@ -0,0 +1,113 @@
+### Lesson_4 - Mailroom Part 2
+
+from textwrap import dedent
+import tempfile
+import sys
+
+donor_db = {'Jimmy Hendrix' : [23.23, 24.48],
+            'Jack White' : [32.84, 48],
+            'Keith Richards' : [68],
+            'Jimmy Page' : [34, 89],
+            'Albert Hamond' : [34, 64, 49],
+            }
+
+def main():
+    prompt = "\n".join(("Please select from the following options",
+         "1 - Send a Thank You",
+         "2 - Create a Report",
+         "3 - Thank All",
+         "4 - Quit",
+         ">>> "))
+    while True:
+        response = input(prompt)
+        if response == "1":
+            thank_you()
+        elif response == "2":
+            report()
+        elif response == "3":
+            thank_all()
+        elif response == "4":
+            break
+        else:
+            print("Please select a number between 1 and 4")
```

Comment:
you should be using a switch dict in this lesson, however I see that you've applied that in lesson 05 so that will work.